app/doc/responses.go

```diff
@@ -18,7 +18,6 @@ type updatedResponse struct {
 }
 
 // CreatedResponse returned when the request has successfully created the object
-// swagger:response updatedResponse
```

Comment:
if it could be used alone, you can leave the `// swagger:response updatedResponse`

Reply:
If I leave it (swagger:response CreatedResponse), the validator warns (because it is unused)